depth_first_traversal/DepthFirstTraversal.java

```diff
@@ -0,0 +1,38 @@
+class DepthFirstTraversal {
+
+    public static int[] arr;    // DFS traversal path
+    public static int k = 0;
```

Comment:
I don't know how but the codacy running here is not mine's but @sandeep201451066 . Maybe since all members have full access to this repo, something got screwed up.